src/AddTodoForm.js

```diff
@@ -1,21 +1,32 @@
 
-import React from 'react';
+import React, {useState} from 'react';
+
 
 export default function AddTodoForm({ onAddTodo }) {
 
+    const [todoTitle, setTodoTitle] = useState('')
+
+    function handleTitleChange (e){
+        const form = new FormData(document.getElementById("form"));
+        const newTodoTitle = form.get("title");
+        setTodoTitle(newTodoTitle)
```

Comment:
This actually works (💯 ) but you can get the most recent title value from the event `e`.

You did below in the `handleAddTodo` function before